TicTacToe/TicTacToe.js

```diff
@@ -0,0 +1,157 @@
+class Game {
+    moves = 0;
+    user = 'X';
+    board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
+    messages = "Game - START!";
+    complete = false;
+
+    reset() {
+        this.moves = 0;
+        this.user = 'X';
+        this.board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
+        this.messages = "Game - START!";
+        this.complete = false;
+    }
+
+    getMoves() {
+        return this.moves;
+    };
+
+    displayBoard() {
+        let text = '';
+        for (let i = 0; i < 3; i++) {
+            for (let k = 0; k < 3; k++) {
```

Comment:
This inner loop can be replaced with `text += this.board[i].join(' ')`. If you're clever, you can also use this trick on the outer array too.